src/containers/Transactions/index.js

```diff
@@ -43,31 +36,36 @@ class RecentTransaction extends Component {
     convertFromBtcToUsd: func,
     convertFromBitsToUsd: func,
     convertSatoshiToBits: func,
-    convertFromUsdToBtc: func
+    convertFromUsdToBtc: func,
+    vendor: bool
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      Transactions: null
+    }
   }
 
   componentDidMount() {
-    const { isEditFormOpen } = this.props
-    isEditFormOpen && this.hideEditForm()
+    this.refreshTransactions(this.props.vendor)
+  }
+
+  refreshTransactions(vendor = false) {
+    this.setState({
+      Transactions: withTransactions(TransactionLog)
```

Comment:
state key name with a capital? Same for `const Transaction` below. These are maybe notes for future.